protocol/0028-GOVE-governance.md

```diff
@@ -679,15 +691,30 @@ It is NOT possible to submit a governance proposal where the source account is t
 
 ##### Batch Proposals
 
-- A batch proposal containing one or more component submissions for each type of proposal term can be submitted and is accepted as a valid proposal. (<a name="0028-GOVE-146" href="#0028-GOVE-146">0028-GOVE-146</a>)
-
-- A batch proposal containing component submissions with different categories will be rejected with an informative error message. (<a name="0028-GOVE-147" href="#0028-GOVE-147">0028-GOVE-147</a>)
+- A batch proposal containing one or more component submissions mixing different proposal types can be submitted and is accepted as a valid proposal. (<a name="0028-GOVE-146" href="#0028-GOVE-146">0028-GOVE-146</a>)
 
-- A batch proposal submitted with component submissions having the same category but different closing timestamps will be rejected with an informative error message. (<a name="0028-GOVE-148" href="#0028-GOVE-148">0028-GOVE-148</a>)
+- A batch proposal submitted with component submissions having different closing timestamps will be rejected with an informative error message. (<a name="0028-GOVE-148" href="#0028-GOVE-148">0028-GOVE-148</a>)
 
-- A batch proposal submitted with component submissions having the same category and the same closing timestamps but different enactment timestamps will be accepted and move to voting.  (<a name="0028-GOVE-149" href="#0028-GOVE-149">0028-GOVE-149</a>)
+- A batch proposal submitted with component submissions with different enactment timestamps will be accepted and move to voting.  (<a name="0028-GOVE-149" href="#0028-GOVE-149">0028-GOVE-149</a>)
    1. If this proposal is accepted, each of the components will be enacted at the time of their differing enactment timestamps. (<a name="0028-GOVE-145" href="#0028-GOVE-145">0028-GOVE-145</a>)
 
+- A batch proposal containing
+   1. freeform proposal,
+   1. an update asset proposal changing the asset quantum, withdrawal delay and withdrawal delay threshold
+   1. a network parameter change,
+   1. a market proposal,
+   1. a change proposal for another market,
+   1. volume discount program,
+   1. referral program,
+   1. governance transfer,
+
+can be submitted, voted through and each proposal enacted.
+On top of that signed bundles for changing withdrawal delay and threshold on the bridge are emitted (<a name="0028-GOVE-160" href="#0028-GOVE-160">0028-GOVE-160</a>)
+
+- A batch proposal can be submitted changing the same network parameter twice to two different values with two different enactment timestamps.
```

Comment:
We should also have one changing the same param twice with the same timestamp. It should pass and the later value is applied if valid. (Because of the principle not to add extra validation for batches.)

Reply:
@barnabee it originally said earlier in that spec that:
> The enactment order of two proposals in the batch with the same enactment timestamp does not need to be defined and should be considered indeterminate from a user's point-of-view.

That has been removed in this current version, but if what you have said is correct I believe it is worth adding an acceptance criteria that proposals in the same batch with the same enactment timestamp are executed in the order specified in the submission.